### ISSUE TITLE
Getting firefox jabref extension to work

### ISSUE THREAD

**Issue by State-Of-Joshing-Gentle-Peevishness** [flathub/org.jabref.jabref#15]
Hei, I hope this is the correct place for my issue. I have installed jabref 5.2 as flatpak. I now want the browser extension for firefox https://docs.jabref.org/collect/jabref-browser-extension. For the snap version are special connect commands, but how about the flatpak version? The file org.jabref.jabref.json I downloaded and placed as per instructions in the mozilla folder is looking for the file "/opt/jabref/lib/jabrefHost.py" but that doesn't seem to be at that path when the flatpak version is installed? Where is that file, I cannot find it? 
sudo find / jabrefHost.py 
find: ‘jabrefHost.py’: No such file or directory
Running on Linux Mint

**Comment by LyzardKing** [comment]
For now to use the flatpak version you have to manually download a file and save it where you want. The file is https://raw.githubusercontent.com/JabRef/jabref/master/buildres/linux/jabrefHost.py

After that you need to create in .mozilla/native-messaging-hosts a file called `org.jabref.jabref.json` and in it copy the following, replacing the path with the one in which you saved the script:

```
{
  "name": "org.jabref.jabref",
  "description": "JabRef",
  "path": "path to the file jabrefHost.py",
  "type": "stdio",
  "allowed_extensions": [
    "browserextension@jabref.org",
    "@jabfox"
  ]
}

```

**Comment by LyzardKing** [comment]
I'll build a script to automate this, then figure out a way to include it in the flatpak..

**Comment by State-Of-Joshing-Gentle-Peevishness** [comment]
Awesome, this worked! If anyone else needs this workaround, the jabrefHost.py script needs to be executable.

**Comment by Jesse-Millwood** [comment]
Has anyone tried with Firefox also as a flatpak? 
I tried putting an executable version of that jabrefHost.py in a location under my home directory. The jabref plugin still says "Error while sending to JabRef". When I go to the JabRef extension's preferences it says 
Status: Attempt to post message on disconnected port
Websocket status: client not started
I did put the json file in my `$HOME/.mozilla` directory. I'm not sure if the flatpak Firefox uses a different directory.
